modules/engines/engine-core/src/main/java/com/sap/xsk/hdb/ds/facade/XSKHDBCoreFacade.java

```diff
@@ -130,9 +136,25 @@ public void updateEntities() {
                 sorted.addAll(dataStructureTableFunctionsModel.keySet());
                 sorted.addAll(dataStructureSchemasModel.keySet());
                 sorted.addAll(dataStructureSynonymModel.keySet());
+                sorted.addAll(dataStructureSequencesModel.keySet());
 
                 boolean hdiOnly = Boolean.parseBoolean(Configuration.get(IXSKEnvironmentVariables.XSK_HDI_ONLY, "false"));
                 if (!hdiOnly) {
+                    //drop HDB Sequences
+                    List<XSKDataStructureHDBSequenceModel> hdbSequencesToUpdate = new ArrayList<>();
+                    for (int i = sorted.size() - 1; i >= 0; i--) {
+                        String dsName = sorted.get(i);
+                        XSKDataStructureHDBSequenceModel model = (XSKDataStructureHDBSequenceModel) dataStructureSequencesModel.get(dsName);
+                        if (model != null) {
+                            hdbSequencesToUpdate.add(model);
+                        }
+                    }
+
+                    IXSKDataStructureManager<XSKDataStructureModel> xskSequencesManagerService = managerServices.get(IXSKDataStructureModel.TYPE_HDB_SEQUENCE);
+                    for (XSKDataStructureHDBSequenceModel sequenceToUpdate :
+                            hdbSequencesToUpdate) {
+                        xskSequencesManagerService.dropDataStructure(connection, sequenceToUpdate);
```

Comment:
we should not drop the sequences, right?

Reply:
Already commented.